src/Components/Links.css

```diff
@@ -30,3 +30,14 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+.w-70 {
```

Comment:
Don't need this. Can be done with build in prime react classes
https://www.primefaces.org/primeflex/gridsystem
https://www.primefaces.org/primeflex/width

Reply:
just tried using p-grid and p-col 
width is not getting adjusted properly based on screen size extra class is needed for this.
After changing to grid style -> (need to limit the max size based on responsiveness)
![image](https://user-images.githubusercontent.com/35369843/142009389-41cca5db-7ba7-4c4f-9bac-5f402819d197.png)

Also could you tell me why prime react class would be better choice instead of normal css classes?

Reply:
I guess its fine. Prime react doesn't seem to have % based widths. And its always better to use a css library classes that we are already using in our project instead of adding css on our own. CSS is hard to maintain and having duplicate CSS only increases the problem. Also we will be shipping extra css into the client that uses this app, which is never good.

Reply:
oohh ok. 
Thanks for letting me know. Will keep this in mind. 😄